pkg/controller/node/nodecontroller.go

```diff
@@ -112,6 +113,11 @@ type NodeController struct {
 	nodeStore      cache.StoreToNodeLister
 
 	forcefullyDeletePod func(*api.Pod)
+	availableCIDRs      sets.Int
+	// Record the first CIDR which has not been put into the availableCIDR set
+	firstAvailable int
```

Comment:
Yup, I understand what it does. I only mean that it should be called firstNotAvailable, firstUnassigned or something along those lines. firstAvailable made me think it's first CIDR to be assign to a node, which is not the case.